.github/workflows/cron.yaml

```diff
@@ -1,8 +1,8 @@
 name: daily
```

Comment:
While you're at it, you could probably change the title of the PR this creates to something more descriptive than the default `Changes by create-pull-request action`.

Reply:
I'd definitely accept a change for that, but I don't want to block on it. I'm going ahead and merging.